lib/serve.js

```diff
@@ -51,6 +52,15 @@ const serve = (watchPath, options) => new Promise((resolve, reject) => {
 
           form.on('file', (field, file) => {
             files.push([field, file]);
+            var data = {
+              'aframe': {
+                'videos': [{
+                  'src': 'assets/video/' + file.name,
+                  'type': file.type
+                }]
+              }
+            };
+            manifest.merge(watchPath + '/package.json', data);
```

Comment:
it might be nice to return this function, since it's a Promise, somewhere so we can block on serving *until* manifest has been updated and written to disk. good good?